src/main/kotlin/com/toasttab/pulseman/view/UserFeedbackUI.kt

```diff
@@ -38,18 +40,25 @@ import com.toasttab.pulseman.state.UserFeedback
  * TODO get selectable text working in this view
  */
 @Composable
-fun userFeedbackUI(state: UserFeedback) {
+fun userFeedbackUI(
+    userFeedback: String,
+    onUserFeedbackClear: () -> Unit
+) {
     Row {
         TextField(
             label = { },
-            value = state.userFeedback.value,
+            value = TextFieldValue(
+                text = userFeedback,
+                // Force the userfeedback to always show the end of the text
+                selection = TextRange(userFeedback.length)
+            ),
```

Comment:
This is really frustrating, I've had to keep using a TextRange to always show the end of the log when a line is added, but because the selection is set to the end of the text you can't copy and paste error text out, I need to revisit this and get a better approach